### PR TITLE
enable option to make login quiet

### PR DESCRIPTION
**Pull request by marcelobartsch**
This change allow to hidden the 'credentials are not expired skipping' message.

This is useful for me as I use the saml2aws in my oh-my-zsh setup everytime before I run a command, and if the credentials are expired, they will get updated. and the message above is a bit annoying to see in the prompt all the times.

### CODE REVIEW

**Comment by RichiCoder1** [comment]
:+1: to this

File: cmd/saml2aws/main.go
Comment (RichiCoder1) [inline cmd/saml2aws/main.go]: May this should just be "Suppresses information messages" as it'd be neat if this could silence the `login` output if you `--skip-prompt`.

Comment (marcelobartsch) [reply cmd/saml2aws/main.go]: Fully Agree on the message.
On the --skip-prompt. when login is required, I'm not sure is a good idea, at least I would leave the 'Waiting for approval, please check your XXXX Verify app' message visible.

Comment (marcelobartsch) [reply cmd/saml2aws/main.go]: Changes done, please my apologies if the way is not the best, I'm not a programmer just a DevOps/SRE/SysAdmin :)
hopefully it is good enough.

**Comment by wolfeidau** [comment]
I really like the idea of this, I am just not super keen on the implementation.

 It is linked to #32 which needs to do different things if we are on Linux and Windows, I really need something which finds all fmt calls and replace them with a global writer which has a quite mode.

Thoughts on this?

**Comment by oshaughnessy** [comment]
@marcelobartsch, I did something similar in #605. Does that take care of your need, too?

**Review by mapkon — CHANGES_REQUESTED** [review]
@marcelobartsch Is this still required? If yes, can you update to achieve parity with master with regards to conflicts and test coverage?

**Pull request closed without merging**
